source/_components/opnsense.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: "OPNsense"
+description: "Instructions on how to configure OPNsense integration"
+logo: opnsense.png
+ha_category:
+  - Hub
+  - Presence Detection
+ha_release: 0.100
+redirect_from:
```

Comment:
Please remove the `redirect_from`

Reply:
Done in https://github.com/home-assistant/home-assistant.io/pull/10427/commits/dc933232ab654d79a27cc0a16f8ca13b16196d46